open-telemetry/opentelemetry-php
# Feature - Implemented exporter factory and added unit tests

# Description
This PR addresses issue #184. With this update the exporter can be selected via configuration string. The implementation was done by creating a factory class called ExporterFactory that takes in a configuration string and creates the correct Exporter after parsing. Prior to this change, exporters needed to be injected to the processors manually as instances.
# Type of change
New feature
# Testing
Unit tests were added to:
- Verify correct exporter was returned
- Test invalid input
- Verify the handling of inputs that don't contain a licenseKey; which should.
# Documentation
Documentation was added to the method 'fromConnectionString' in the ExporterFactory class.
